Media/DisplayMedia/Strategies/AbstractStrategy.php

```diff
@@ -58,8 +58,14 @@ public function displayPreview(MediaInterface $media)
      */
     public function displayMediaForWysiwyg(MediaInterface $media, $format = '')
     {
-        return '<img class="tinymce-media" src="' . $this->getFileUrl($media->getThumbnail()) . '" alt="'
-            . $media->getAlt($this->request->getLocale()) . '" data-id="' . $media->getId() . '" />';
+        return $this->container->get('templating')->render(
```

Comment:
@nicolasThal : I'm not fan of this solution, because doing this you introduce a two-ways dependancy:
1/ The manager knows all its strategies due to the compiler pass (that one is correct)
2/ The strategies knows the manager via the setManager method (that one annoys me)